File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    s = s.lower()
    
    s = list(s)
    
    s[0] = s[0].upper()
    
    for i in range(0, len(s)):
        if(s[i] == ' '):
            if(i < len(s)-1):
                s[i+1] = s[i+1].upper()
    
    return ''.join(s)

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
  s = s.toLowerCase();
  s = s.split('');
  s[0] = s[0].toUpperCase();
  for (i = 0; i < s.length; i++) {
    if (s[i] === ' ') {
      if (i < s.length - 1) {
        s[i + 1] = s[i + 1].toUpperCase();
      }
    }
  }
  return s.join('');
}